Core/include/Acts/EventData/ParameterConcept.hpp

```diff
@@ -89,10 +92,12 @@ namespace concept {
         has_method<const P, FreeVector, parameters_t>;
 
     /// Tie all free results together
-    constexpr static bool freeValue =
-        require<free_position_exists, free_momentum_exists, free_charge_exists,
-                free_time_exists, free_covariance_exists,
-                free_parameters_exists>;
+    constexpr static bool freeValue = require<free_position_exists,
+                                              free_momentum_exists,
+                                              free_charge_exists,
+                                              free_time_exists,
+                                              free_covariance_exists,
+                                              free_parameters_exists>;
```

Comment:
Just commenting so it shows up as an example how it would look before/after the change.

Reply:
Thanks. I hate it 😄